docs/tutorials/subcommands.md

```diff
@@ -1,15 +1,44 @@
 # Using Subcommands
 
-**TODO** Better title?
+The document gives an overview of the subcommands that beaker supports and
+describes a basic of how to use them.
 
 ## Why Subcommands?
 
+Subcommands are designed to make test development and iteration simpler by
+separating out all of the phases of a beaker test run*. Instead of requiring the
```

Comment:
@tvpartytonight for reference: our [Test Run doc](https://github.com/puppetlabs/beaker/blob/master/docs/tutorials/test_run.md).